docs/_data/toc.yml

```diff
@@ -168,6 +168,8 @@
         url: guides/tutorials/exploring-kubernetes-cronjobs
       - title: Understanding Kubernetes ConfigMaps and Secrets
         url: guides/tutorials/kubernetes-configmaps-secrets
+      - title: Hello Kubernetes with Dapr
+        url: guides/tutorials/Hello-Kubernetes-with-Dapr
```

Comment:
Lowercase lettering, you shall use.

There, I said it like Yoda, so maybe you'll understand this time.